internal/configs/parser_config_dir.go

```diff
@@ -111,6 +168,13 @@ func (p *Parser) dirFiles(dir string) (primary, override []string, diags hcl.Dia
 			continue
 		}
 
+		if ext == ".tftest" || ext == ".tftest.json" {
```

Comment:
Why do we handle `.tftest.json` here, but not in `fileExt` below?

Reply:
This was a bug, fixed and added a test that loads .tftest.json files to verify the fix.